### ISSUE TITLE
TeeInputHasResult.java:46-48: Delete TeeInputHasResult...

### ISSUE THREAD

**Issue by 0pdd** [llorllale/cactoos-matchers#68]
The puzzle `58-b44e1a44` from #58 has to be resolved: 

https://github.com/llorllale/cactoos-matchers/blob/63681b90b3e137fa5986a744a3b5cd0b14dc898e/src/main/java/org/llorllale/cactoos/matchers/TeeInputHasResult.java#L46-L48

The puzzle was created by @rultor on 03-Jan-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0pdd** [comment]
This puzzle inherited milestone `1.0.0` from issue #58.

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#68](https://github.com/llorllale/cactoos-matchers/issues/68) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#68](https://github.com/llorllale/cactoos-matchers/issues/68) assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/5a5e20ee-9e7e-4181-a8d6-26649a48b032); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by llorllale** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/llorllale/cactoos-matchers/issues/68#issuecomment-474140252))

@llorllale The user @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) resigned from [#68](https://github.com/llorllale/cactoos-matchers/issues/68), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
The job [#68](https://github.com/llorllale/cactoos-matchers/issues/68) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/572217ba-ff9e-4e23-931f-f38d7641cfd0); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by victornoel** [comment]
@llorllale what does `Should be moved to test packages in cactoos` mean here? What should be moved to cactoos?

**Comment by victornoel** [comment]
@llorllale see my comment above

**Comment by llorllale** [comment]
@victornoel good question. Not sure what I was thinking at the time.

If "users should use `InputHasContent`", then `TeeInputHasResult` should not be moved to cactoos at all - it should be deleted and replaced by `InputHasContent`.

Looks to me like all that's left to do in `cactoos-matchers` is to delete `TeeInputHasResult`. `cactoos` will have to refactor later on when upgrading version.

**Comment by victornoel** [comment]
@0crat wait for #104 to be merged.

@llorllale once #104 is merged I think we can release a new version and I can take care of updating cactoos-matchers in cactoos as part of this issue since there wasn't so much work here to do. Or we can wait for #103 maybe, you tell me :)

**Comment by 0crat** [comment]
> @0crat wait for #104 to be merged. @llorllale once #104 is merged I think we can release a new... ([here](https://github.com/llorllale/cactoos-matchers/issues/68#issuecomment-479162603))

@victornoel The impediment for [#68](https://github.com/llorllale/cactoos-matchers/issues/68) was registered successfully by @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0pdd** [comment]
The puzzle `58-b44e1a44` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#68](https://github.com/llorllale/cactoos-matchers/issues/68) is now out of scope

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)
